site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -145,6 +145,69 @@
     <br/>
     <hr>
 
+    <div class="dropdown-wrapper">
+        <div id="tomorrow-table-dropdown" onclick="updateDropdownStates($(this).find('.dropdown-arrow').hasClass('down'), 'tomorrow_polls_dropdown', '{{ base_url }}')" class="dropdown-bar">
+            <h2> Tomorrow's Polls </h2>
+            <div class="dropdown-btn">
+                <i class="dropdown-arrow{{dropdown_states['future'] ? ' down' : ' right'}} fa fa-chevron-right"></i>
```

Comment:
I believe this line is the source of the problem.  The "tomorrow's polls" dropdown matches the state of the "future polls" dropdown.

Reply:
Hard to say if it is an intended feature by the old PR or not. Anyways, I added one more dropdown status so now the "tomorrow's" dropdown doesn't listen to the status of other dropdowns, and made it open by default just like "Today's Polls". I brought up the order of "Tomorrow's Polls" to be directly under "Today's" to make it more accessible to the user. 

Now the order of the dropdowns goes like this: 
Today's => Tomorrow's => Old => Future
I'd think it makes sense to swap the order of "Old Polls" and "Future Polls" dropdowns as well.